src/components/EditGrower.js

```diff
@@ -38,25 +38,34 @@ const EditGrower = (props) => {
   const [growerUpdate, setGrowerUpdate] = useState(null);
   const [loadingGrowerImages, setLoadingGrowerImages] = useState(false);
   const [saveInProgress, setSaveInProgress] = useState(false);
-
+  const [customImageUrl, setCustomImageUrl] = useState(null);
+  const [customImageUrlError, setCustomImageUrlError] = useState({
+    label: '',
+    error: false,
+  });
+  const regex = /\.(gif|jpe?g|tiff?|png|webp|bmp)$/i;
   useEffect(() => {
     async function loadGrowerImages() {
       if (grower?.id) {
         setLoadingGrowerImages(true);
         const selfies = await growerContext.getGrowerSelfies(grower.id);
         setLoadingGrowerImages(false);
-
         setGrowerImages([
-          ...(grower.imageUrl ? [grower.imageUrl] : []),
-          ...(selfies || []).filter((img) => img !== grower.imageUrl),
+          customImageUrl === '' ? null : customImageUrl,
+          ...(grower.image_url ? [grower.image_url] : []),
+          ...(selfies || []).filter((img) => img != grower.image_url),
         ]);
       }
     }
-
     setGrowerUpdate(null);
     loadGrowerImages();
-  }, [grower]);
-
+  }, [grower, customImageUrl]);
```

Comment:
Including `customImageUrl` here as a trigger here causes us to reload images and reset the dialog, meaning the custom image is not auto-selected.

Instead, just add the custom URL to the list of images when you pass it to `ImageScroller`:
```js
          <ImageScroller
            images={[
              customImageUrl === '' ? null : customImageUrl,
              ...growerImages
            ].filter((e) => e !== null)}
```